frontend/src/core/observations/ObservationBulkAssessment.tsx

```diff
@@ -137,7 +137,7 @@ const ObservationBulkAssessment = (props: ObservationBulkAssessmentButtonProps)
                                 choices={OBSERVATION_VEX_JUSTIFICATION_CHOICES}
                             />
                         )}
-                        <TextInputWide source="comment" validate={validate_required_255} />
+                        <TextInputWide source="comment" validate={validate_required} />
```

Comment:
```suggestion
                        <TextInputWide source="comment" validate={validate_required} minRows={3} />
```

and see remark about the size limitiation in the model